mysite/settings/conf.py

```diff
@@ -1,559 +0,0 @@
-        },
-
-        # sorl-thumbnail settings:
-        'THUMBNAIL_CACHE': 'thumbnails',
-        'THUMBNAIL_DEBUG': debug,
-
-        # Settings for restricting user activity to reduce abuse:
-        'RESTRICT_RENAMES': conf.get('RESTRICT_RENAMES'),
-        'EDITS_ALLOWED': conf.get('EDITS_ALLOWED', True),
-        'SHOW_BANNER': conf.get('SHOW_BANNER', False),
-
-        # A bearer token for the Twitter API for mapping between
-        # Twitter usernames and IDs.
-        'TWITTER_APP_ONLY_BEARER_TOKEN': conf.get('TWITTER_APP_ONLY_BEARER_TOKEN'),
-
-        # Django Rest Framework settings:
-        'REST_FRAMEWORK': {
-            'DEFAULT_PERMISSION_CLASSES': ('candidates.api_permissions.ReadOnly',),
-            'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.URLPathVersioning',
-            'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',),
-            'DEFAULT_RENDERER_CLASSES': (
-                'rest_framework.renderers.JSONRenderer',
-                'rest_framework_jsonp.renderers.JSONPRenderer',
-                'rest_framework.renderers.BrowsableAPIRenderer',
-            ),
-            'PAGE_SIZE': 10,
-        },
-
-        # allow attaching extra data to notifications:
-        'NOTIFICATIONS_USE_JSONFIELD': True,
-
-        'HAYSTACK_SIGNAL_PROCESSOR': 'haystack.signals.RealtimeSignalProcessor',
-
-        'HAYSTACK_CONNECTIONS': {
-            'default': {
-                'ENGINE': 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine',
-                'URL': 'http://127.0.0.1:9200/',
-                'INDEX_NAME': '{0}_{1}'.format(conf.get('YNMP_DB_NAME'), conf.get('YNMP_DB_HOST')),
-            },
-        },
-
-        # CORS config
-        'CORS_ORIGIN_ALLOW_ALL': True,
-        'CORS_URLS_REGEX': r'^/(api|upcoming-elections)/.*$',
-        'CORS_ALLOW_METHODS': (
-            'GET',
-            'OPTIONS',
-        ),
-
-        'MARKDOWN_DEUX_STYLES': {
-            "default": {
-                "extras": {
-                    "code-friendly": None,
-                    "fenced-code-blocks": None,
-                },
-                "safe_mode": "escape",
-            },
-        }
-    }
-    if not conf.get('NEW_ACCOUNTS_ALLOWED', True):
-        result['ACCOUNT_ADAPTER'] = \
-            'mysite.account_adapter.NoNewUsersAccountAdapter'
-    result['CANDIDATES_REQUIRED_FOR_WEIGHTED_PARTY_LIST'] = \
-        conf.get('CANDIDATES_REQUIRED_FOR_WEIGHTED_PARTY_LIST', 20)
-    result['HOIST_ELECTED_CANDIDATES'] = \
-        conf.get('HOIST_ELECTED_CANDIDATES', True)
-    if tests:
-        result['NOSE_ARGS'] = [
-            '--nocapture',
-            '--with-yanc',
-            # There are problems with OpenCV on Travis, so don't even try to
-            # import moderation_queue/faces.py
-            '--ignore-files=faces',
-        ]
-        if election_app == 'example':
-            result['NOSE_ARGS'].append('--with-doctest')
```

Comment:
You've removed the `--with-doctest` arg here. I know we don't tend to use them very much, but it looks like there are at least some doctests in this project e.g: https://github.com/DemocracyClub/yournextrepresentative/blob/06e5752a3906d57d95af97b4eaffd7b62757b3c3/moderation_queue/views.py#L180-L188 which I suspect are now being ignored?